src/App.js

```diff
@@ -30,15 +30,15 @@ class App extends Component {
         <Nav />
         {this.state.error && <Error error={this.state.error} />}
         {!this.state.movies.length && !this.state.error && <h1>Loading...</h1>}
-        <Route exact path='/' render={({ location }) => {
-          return(<MoviesView movieList={this.state.movies} />)
+        <Switch>
```

Comment:
Nice work figuring out the switch and updating the path so that it works as expected